### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by yahyaqowle** [yahyaqowle/Afsoomali#11] [General]
Don't do B words. I'm Finishing B words today total of 25 pages. I Want you to create a new text-file and name it "C words.txt" in the same folder we were working on and do your one page there.

**Comment by yahyaqowle** [comment]
Also @haith4m try to give any updates here. it will be good if you are active here and give a report/summary.

**Reply by haith4m** [reply]
Okay thanks

**Reply by haith4m** [reply]
Will do right now

**Reply by yahyaqowle** [reply]
i'm sorry akhi, i had some errands to do. will you be available tonight 21:00-22:00 my time, if not let's schedule.

**Reply by haith4m** [reply]
Asc walaal, it's been a while 😅. Ciid mubarak, how have you been. I just remembered about this project, I am now in university studying and eager to make the website(Somali version of dictionary.com). insha'Allah you are well. I want to thank you walahi for giving me this big opportunity, it was my first tech project and alhamdullilah I am now a computer science student.

**Reply by yahyaqowle** [reply]
Aleykum selam akhi, yeah it's been a long time and I'm doing well alhamdulilah. Go ahead and do it. İ will be here for you and I'm  excited to hear that you're on your way to your goal. Update me and I don't know if you know but i finished 44k words and all of it are in the repo. ☺️

**Reply by haith4m** [reply]
Wow that's amazing news brother, so glad to hear. May Allah reward you for your hard work ameen.

**Comment by yahyaqowle** [comment]
@haith4m well done akhi, we have done great job today, i will be travelling for few days so while i'm busy i will be glad if you keep doing the one pages and i will be approving the pull requests from you. After i come back we will schedule a meeting insha'allah. hbeen wngsan.

**Reply by haith4m** [reply]
In sha Allah brother. I may be able to make it two or three pages a day if you want.

**Reply by yahyaqowle** [reply]
İf so it would be even better.

**Reply by haith4m** [reply]
Ok great!

**Comment by yahyaqowle** [comment]
Don't do any C words. I want to finish C words today or tomorrow, i'm sorry i haven't given any updates yesterday. But instead You can Start D words while i'm finishing C words. D words is 118 pages so it's better if you start early and put atleast 5 pages a day and mine will be 10 pages a day, with that numbers we can finish in a week. But i think before you start D words it's better if we can have meeting. i'm available all day.

**Reply by haith4m** [reply]
Okay got it. We can have a meeting at around 3 30 pm. Im not sure ifmy family have any plans today because its jimce but I'll let you know if anything changes.

**Reply by haith4m** [reply]
I can't do it today sorry, but I will be able to tomorrow at the same time in sha Allah.

**Reply by yahyaqowle** [reply]
Tomorrow 15:30 your time or my time?

**Reply by haith4m** [reply]
My time would that be 17:30 for you?

**Reply by yahyaqowle** [reply]
Yes

**Comment by yahyaqowle** [comment]
Today i will be doing page 187(164) till page 200(177) insha allah.

**Reply by haith4m** [reply]
In sha Allah. I will start from 202.

**Reply by yahyaqowle** [reply]
201 you mean right

**Reply by haith4m** [reply]
Yh

**Reply by yahyaqowle** [reply]
How is it going? which page are u in now and have u finished yet?

**Comment by yahyaqowle** [comment]
I will finish the D words in 2 days. While i'm finishing D words can you start E words. E words are few i need you to create and Finish all of it in the next 2 days. Also i moved to dormitory and my internet aint good sorry about that if don't respond early.

**Comment by haith4m** [comment]
No problem. Im starting College on Monday so I will try to finish it by tomorrow.

**Comment by yahyaqowle** [comment]
Asc wll @haith4m, i'm learning how i can work different branch and not use main branch when i'm commiting(doing changes), it's easier i guess to not create problems/bugs and also it's better for us to get used to it since it's a must to know git/github well. So i guess you also need to research more. Good night.

**Reply by haith4m** [reply]
I've been kinda busy because of college but I will try my best to learn more in sha Allah and pls tell me when u resolve the issue so i can begin pushing again.

**Reply by yahyaqowle** [reply]
Take a one week break mate, my uni will start tomorrow and i will finish F and G words. If you want you can start H words and start pushing.

**Comment by haith4m** [comment]
No problem. I will try.

**Comment by yahyaqowle** [comment]
Asc wll, sorry i haven't done progress on the project. İ was busy with my uni and i started distance learning of a second üniversity on compute programming. What i have done so far is creating a simple database and i will share with you soon.

**Reply by haith4m** [reply]
No problem

**Reply by yahyaqowle** [reply]
here is the link https://qaamuus-afsomali.000webhostapp.com/dictionary.php

if u r using chrome it says the website is dangerous, just click continue and if it cant connect reload the page. i used 000webhost and they dont provide ssl certificate

**Reply by yahyaqowle** [reply]
> here is the link https://qaamuus-afsomali.000webhostapp.com/dictionary.php
> 
> if u r using chrome it says the website is dangerous, just click continue and if it cant connect reload the page. i used 000webhost and they dont provide ssl certificate

the  error message "This site can’t provide a secure connection"

**Reply by haith4m** [reply]
Mashallah wll, that's amazing.
What did you do?

**Reply by yahyaqowle** [reply]
> Mashallah wll, that's amazing.
> What did you do?

I created a database and php code and linked to a host

**Reply by haith4m** [reply]
That sounds really useful. Is it easier than making a list like before.

**Reply by yahyaqowle** [reply]
İndeed. But im looking another solution now. İ will tell u the details when we get a meeting. Also can u take a look vocabulary.com and check if you can do something that look their design.

**Reply by haith4m** [reply]
Im not that good at css but I will watch some videos that show you how to make a similar website.
